### PR TITLE
#157440623 Design auth page

### PR DESCRIPTION
**Pull request by bahdcoder**
#### What does this PR do?

Designs the authentication screen.

#### Description of Task to be completed?
 - add AuthPage CSS styles
 - add assets for authentication screen images and fonts
 - update stylelintrc.json file with rules from wiki

#### How should this be manually tested?

Visit the authentication screen

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#157440623](https://www.pivotaltracker.com/story/show/157440623)

#### Questions:
None

#### Screenshots
screen shot 2018-06-04 at 12 22 31 pm

### CODE REVIEW

**Comment by andela-kakpobome** [comment]
@bahdcoder edit your PR title to be in sentence case i.e `#157440623 Design auth page`

File: src/pages/AuthPage/AuthPage.scss [hunk 1]
```diff
@@ -1,5 +1,92 @@
...
 
-.h1-tag {
-  color: map-get($grey , 100);
+$boxshadow: 0.0625rem 0.3125rem 1rem 0 rgba(0, 0, 0, 0.19);
+
+button {
+  &:active {
+    outline: none;
+  }
+}
```
Comment (andela-kakpobome) [inline src/pages/AuthPage/AuthPage.scss]: This should be in the globals file, it should apply globally.

File: src/pages/AuthPage/AuthPage.scss [hunk 2]
```diff
@@ -1,5 +1,92 @@
 @import '../../assets/scss/variables';
 
-.h1-tag {
-  color: map-get($grey , 100);
+$boxshadow: 0.0625rem 0.3125rem 1rem 0 rgba(0, 0, 0, 0.19);
```
Comment (andela-kakpobome) [inline src/pages/AuthPage/AuthPage.scss]: Move this to `variables`

File: src/pages/AuthPage/AuthPage.scss [hunk 3]
```diff
@@ -1,5 +1,92 @@
...
+  font-size: 1.5rem;
+  font-weight: 300;
+  line-height: 1.938rem;
+  color: map-get($grey, 100);
+}
+
+.jumbotron {
+  h1 {
+    margin: 16.25rem 0 0 1.063rem;
+    font-family: 'DinPro';
```
Comment (andela-kakpobome) [inline src/pages/AuthPage/AuthPage.scss]: This line is not necessary, since it's globally set.

File: src/pages/AuthPage/AuthPage.scss [hunk 4]
```diff
@@ -1,5 +1,92 @@
...
+
+.login-button {
+  background-color: map-get($grey, 100);
+  width: 14rem;
+  height: 3.125rem;
+  padding: 0;
+  border: 0;
+  border-radius: 0 0.25rem 0.25rem 0;
+  border-left: 0.0625rem solid map-get($palette, primary);
+  font-family: 'DinPro';
```
Comment (andela-kakpobome) [inline src/pages/AuthPage/AuthPage.scss]: Remove this as well.

File: src/pages/AuthPage/AuthPage.scss [hunk 5]
```diff
@@ -1,5 +1,92 @@
...
+    margin: 16.25rem 0 0 1.063rem;
+    font-family: 'DinPro';
+    font-size: 2.25rem;
+    font-weight: 500;
+    line-height: 2.75rem;
+    color: map-get($grey, 100);
+  }
+}
+
+.login {
```
Comment (andela-kakpobome) [inline src/pages/AuthPage/AuthPage.scss]: A better name for this would be `login-btn-container`

File: src/pages/AuthPage/AuthPage.scss (hunk 1 above)
Comment (bahdcoder) [reply src/pages/AuthPage/AuthPage.scss]: This has been resolved.

File: src/pages/AuthPage/AuthPage.scss (hunk 2 above)
Comment (bahdcoder) [reply src/pages/AuthPage/AuthPage.scss]: Resolved.

File: src/pages/AuthPage/AuthPage.scss (hunk 3 above)
Comment (bahdcoder) [reply src/pages/AuthPage/AuthPage.scss]: Resolved.

File: src/pages/AuthPage/AuthPage.scss (hunk 4 above)
Comment (bahdcoder) [reply src/pages/AuthPage/AuthPage.scss]: Resolved.

**Pull request merged**